doc/gallery.rst

```diff
@@ -0,0 +1,58 @@
+ .. role:: html(raw)
+   :format: html
+
+.. _New_Users:
+
+Tutorial gallery
+================
+
+These step-by-step tutorials provided a gentle introduction to the basics of PennyLane. Once you have
+worked your way through the QuickStart tutorials, you will be ready to implement your own
+interesting QML problems in PennyLane!
+
+
+Intermediate tutorials
+----------------------
+
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to rotate a qubit!
+    :figure: tutorials/figures/bloch.png
+    :description: :ref:`qubit_rotation`
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to control a beamsplitter!
+    :figure: tutorials/figures/gaussian_transformation.png
+    :description: :ref:`gaussian_transformation`
+
+.. customgalleryitem::
+    :tooltip: Multiple expectation values, Jacobians, and keyword arguments.
+    :description: :ref:`advanced_features`
+
+:html:`<div style='clear:both'></div>`
+
+Using plugins
+-------------
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to create a qubit rotation/beamsplitter optimization loop!
+    :figure: tutorials/figures/photon_redirection.png
+    :description: :ref:`plugins_hybrid`
+
+.. customgalleryitem::
+    :tooltip: Do arbitrary state preparation on a real quantum computer!
+    :description: :ref:`state_preparation`
+
+.. customgalleryitem::
+    :tooltip: Extend PyTorch with real quantum computing power!
+    :figure: tutorials/figures/bloch.gif
+    :description: :ref:`pytorch_noise`
+
+:html:`<div style='clear:both'></div>`
+
+Examples
+--------
+
+Let's convert the examples folder to sphinx-gallery style, and then
+we can include ising.py here as well!
```

Comment:
```suggestion
we can include ising.py here as well.
```
(sorry, I'm on a roll)

Reply:
The exclamation marks were because I got too excited :laughing: